cmd/limactl/list.go

```diff
@@ -126,7 +149,12 @@ func listAction(cmd *cobra.Command, args []string) error {
 	}
 
 	if goFormat != "" {
-		tmpl, err := template.New("format").Parse(goFormat)
+		funcMap := template.FuncMap{
+			"indent":  indentString,
+			"trim":    trimString,
+			"missing": missingString,
+		}
```

Comment:
Can we move this to under `pkg/`, and use this for `limactl inspect` too?

Reply:
I'd also like to see unit tests

Reply:
Also documentation; not sure where to put it though

Reply:
> Also documentation; not sure where to put it though

Currently the documentation is duplicated in the cmds

`--format '{{ <go template> }}' - if the format begins and ends with '{{ }}', then it is used as a go template.`

Reply:
Maybe something like `--list-fields`, like `--list-funcs` ?

Which by the way seems to be missing from `inspect'